app/views/layouts/_header.html.erb

```diff
@@ -0,0 +1,12 @@
+  <header class="navbar navbar-fixed-top navbar-inverse">
+    <div class="container">
+      <%= link_to "sample app", "#", id: "logo" %> 
+	  <nav>
+        <ul class="nav navbar-nav navbar-right"> 
+	      <li><%= link_to "Home", "#" %></li> 
+	      <li><%= link_to "Help", "#" %></li> 
+	      <li><%= link_to "Log in", "#" %></li>
+        </ul> 
```

Comment:
wrong indent, only have 2 spaces for child tag html

Reply:
<img width="434" alt="header" src="https://user-images.githubusercontent.com/68420253/90092244-fc6cb800-dd5a-11ea-9e95-f13f1e1ef632.png">
may I know what is wrong, I haven't change any line for it but in my laptop it shown only 2 spaces for my child tag html?